cloud-sql-proxy/cloud-sql-proxy.sh

```diff
@@ -313,68 +332,21 @@ function run_validation() {
   fi
 }
 
-function configure_hive_warehouse_dir() {
-  # Wait for master 0 to create the metastore db if necessary.
-  run_with_retries run_validation
-
-  local hiveserver_uri
-  hiveserver_uri=$(get_hiveserver_uri)
-  HIVE_WAREHOURSE_URI=$(beeline -u "${hiveserver_uri}" -e "describe database default;" |
-    sed '4q;d' | cut -d "|" -f4 | tr -d '[:space:]')
-
-  echo "Hive warehouse uri: $HIVE_WAREHOURSE_URI"
-
-  bdconfig set_property \
-    --name 'hive.metastore.warehouse.dir' \
-    --value "${HIVE_WAREHOURSE_URI}" \
-    --configuration_file /etc/hive/conf/hive-site.xml \
-    --clobber
-  echo "Updated hive warehouse dir"
-}
-
-function reload_function() {
```

Comment:
nit: maybe add to commit message that we intend to move this into internal Dataproc startup so it doesn't just seem like we're deleting it for no reason. In the meantime, since we haven't done that yet, would it make sense to keep this around but only run it if hive-server2 is running?

Reply:
Kept it if hive-server2 is running for backward compatibility.